File: example/javalib/testing/6-test-group-stealing/foo/test/src/foo/GroupX3.java
package foo;

import org.junit.Test;

public class GroupX3 extends RandomTestsUtils {
  @Test
  public void test1() throws Exception {
    testGreeting("Fortuna", 25);
  }

  @Test
  public void test2() throws Exception {
    testGreeting("Gaia", 22);
  }

  @Test
  public void test3() throws Exception {
    testGreeting("Helios", 28);
  }

  @Test
  public void test4() throws Exception {
    testGreeting("Iris", 24);
  }
}

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupX3.kt
package foo

import org.junit.jupiter.api.Test

class GroupX3 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Fortuna", 25)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Gaia", 22)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Helios", 28)
    }

    @Test
    @Throws(Exception::class)
    fun test4() {
        testGreeting("Iris", 24)
    }
} 
